sunpy/image/coalignment.py

```diff
@@ -105,7 +105,8 @@ def clip_edges(data, yclips, xclips):
     """
     ny = data.shape[0]
     nx = data.shape[1]
-    return data[yclips[0].value: ny - yclips[1].value, xclips[0].value: nx - xclips[1].value]
+    return data[int(yclips[0].value): ny - int(yclips[1].value),
+                int(xclips[0].value): nx - int(xclips[1].value)]
```

Comment:
The documentation says *ceiling* values, however, when using `int` you are *flooring* the value - should the documentation be updated?:
```python
>>> a = 3.6 * u.pix
>>> print(int(a.value))
3
>>> print(np.ceil(a))
<Quantity 4.0 pix>
```